src/upnp.cpp

```diff
@@ -2,8 +2,9 @@
  * @file src/upnp.cpp
  * @brief todo
  */
-#include <miniupnpc.h>
-#include <upnpcommands.h>
+#define MINIUPNP_STATICLIB
```

Comment:
I can see that only affects Windows, but I'd feel better if it was behind an `#ifdef _WIN32`.

Reply:
Moved to cmake.